src/langs/en.json

```diff
@@ -511,5 +511,9 @@
   "DELETE_GUESTS_MODAL_CONTENT": "I confirm deleting <1>permanently</1> the following readers and their data:",
   "DELETE_GUESTS_MODAL_DELETE_BUTTON": "Delete permanently",
   "ADD_TAG_OPTION_BUTTON_TEXT": "Add {{value}}",
-  "CREATE_FOLDER_BUTTON_TEXT": "Create Folder"
+  "CREATE_FOLDER_BUTTON_TEXT": "Create Folder",
+  "TAGS_DESCRITPION": "Tags are pieces of data that you can attach to your items. They describe the subject, context, project, or intended audience of your items. They help other users to find your content when they search in the library. You can find out more about tags in <1>the documentation</1>.",
```

Comment:
`Tags are pieces of data...` - "pieces of data" sounds very weird. It does not seem clear to me what this description is trying to convey.

The Wikipedia article describes tags as:
_In information systems, a tag is a keyword or term assigned to a piece of information (such as an Internet bookmark, multimedia, database record, or computer file). This kind of metadata helps describe an item and allows it to be found again by browsing or searching. Tags are generally chosen informally and personally by the item's creator or by its viewer, depending on the system, although they may also be chosen from a controlled vocabulary._

Maybe we can reuse this and do something like:
`Tags are keywords or terms that you can attach to your files...` - and the rest can stay the same.